### DISCUSSION TITLE
Module connection sim-900 ?

### DISCUSSION THREAD

**Discussion by sensormatic1985** [taligentx/dscKeybusInterface#223] [dscKeybusInterface Discussion]
good afternoon, it is possible that I can connect this code with a SIM-900 module and an arduino uno
is there any guide to perform this procedure?

**Comment by sensormatic1985** [comment]
Good morning, could you do me a favor and help me with this question?
I thank you very much

**Reply by kricon** [reply]
You can find TinyGSM-SMS Arduino version on my fork under develop branch. However I still didn't tested it so I don't know if it really works, I have SIM800 module so I hope to test in upcoming days. If all went fine, I'll made ESP8266 compatible version aswell and open a PU.

**Reply by sensormatic1985** [reply]
understood thank you very much

**Reply by kricon** [reply]
@sensormatic1985 I've checked and included both Arduino and esp8266 TinyGSM-SMS examples in my fork.
